aten/src/ATen/templates/TensorMethods.cpp

```diff
@@ -110,82 +66,6 @@ bool Tensor::has_names() const {
   return impl::has_names(unsafeGetTensorImpl());
 }
 
-bool is_cuda(Tensor self) {
-  return self.is_cuda();
-}
```

Comment:
As far as I could tell, these free function variants aren't declared in any header or used anywhere in the code base. It's just dead code.